src/ScillaContractDeployer.ts

```diff
@@ -367,7 +373,8 @@ export async function sc_call(
   sc: Contract,
   transition: string,
   args: Value[] = [],
-  callParams: CallParams
```

Comment:
This will break our current test. This parameter lets us override params like nonce and amount.